setup.cfg

```diff
@@ -39,6 +39,7 @@ molecules_extra =
   MDAnalysis
   ipywidgets
   google.colab
+  requests
```

Comment:
Maybe this should live in the default installation, since otherwise it creates a barrier from the simple installation with `pip install inductiva`.

I think it is okay to be there, since it should be a simple package, however verify if it does not cause any problems during installation.

Reply:
why install requests? we have urllib3 already as a dependency, we can use it to download files.

Reply:
Ah good catch! I will do it with urllib.

Reply:
I've fixed it to use urllib.